index.html

```diff
@@ -15,48 +17,35 @@
     </head>
 
     <body>
-        <div class="width-wrap"></div>
+        <div class="width-wrap">
         <header id="top">
             <nav>
                 <ul>
                     <li>
-                        <p>
                             <a href="index.html">Home</a>
-                        </p>
                     </li>
                     <li>
-                        <p>
                             <a href="links.html">Links</a>
-                        </p>
                     </li>
                     <li>
-                        <p>
                             <a href="glossary.html">Glossary</a>
-                        </p>
                     </li>
                 </ul>
             </nav>
         </header>
         <main>
-            <H1>Welcome to Calvin Lane's CSD120 Website!</H1>
+            <h1>Welcome to Calvin Lane's CSD120 Website!</h1>
             <p>
                 My name is Calvin Lane, I am loving this opportunity to learn web design through programming and not a site-builder.
-                <br>
                 I just completed High School at Superior Heights Collegiate and Vocational School and instantly moved into college to further my skills within programming.
-                <br>
                 In my free time I create useless robots with arduino, play electic bass and guitar, and build/fix miscellaneous items.
-                <br>
                 I want to be the kind of person who can give assistance to other people when they need it and try and be as nice as possible when meeting people.
-                <br>
                 I aspire to do well in my classes and have a very enjoyable time in my freshman year of college!
-                <br>
-                <br>
+            </p>
+            <p>
                 I decided to add my second paragraph once I finished validating and fixing my code because I can add my thoughts on HTML as the course passes through!
-                <br>
                 HTML is becoming more enjoyable as I learn the concepts, when I started this assignment I gave the instructions a quick read before doing the research, I will say that it seemd very intimidating.
-                <br>
                 After finishing the code and learning what all of the concepts are and what everything does, I can look back on this as a very positive experience, I quite enjoy the formatting of HTML, its very user friendly.
-                <br>
```

Comment:
Use `<p>` to separate paragraphs, not `<br>`
Don't use `<br>` for line-length formatting